include/aikido/io/util.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef AIKIDO_IO_UTIL_HPP_
+#define AIKIDO_IO_UTIL_HPP_
+
+#include <aikido/io/CatkinResourceRetriever.hpp>
+#include <dart/dart.hpp>
+#include <dart/utils/urdf/DartLoader.hpp>
+
+using dart::dynamics::SkeletonPtr;
```

Comment:
We should avoid `using` in headers because it can pollute namespaces by including this header. Let's remove and use explicitly `dart::dynamics::SkeletonPtr` in headers. You could use `using` in sources.